Libs/GLib/Structs/Error.cs

```diff
@@ -7,14 +7,14 @@ public partial struct Error
     {
         #region Properties
 
-        public string Message => Marshal.PtrToStringAuto(message);
+        public string Message => Marshal.PtrToStringAnsi(message);
```

Comment:
Why is Ansi better compared to Auto?

Reply:
This was a really bizarre issue I found while playing around with CssProvider. Take
the following method as an example (implementation not important):

```cs
// This function loads an embedded stylesheet and creates a CSS
// provider from it.
private static CssProvider CreateGlobalProvider(Assembly asm)
{
    var provider = new CssProvider();

    var cssStr = GetStylesheetFromAssembly(asm);
    provider.LoadFromData(cssStr, out GError error);

    if (error != null)
        throw new Exception(error?.Message); // <-- THIS IS THE RELEVANT PART

    return provider;
}
```

When we marshal `message` as either Auto or Uni, and throw an exception like in the snippet above:

```
Unhandled Exception: System.Exception: ?????????????????????
```

While if we marshal as Ansi:

```
Unhandled Exception: System.Exception: <data>:2:16expected '}' after declarations
```

I have no clue why this happens, but PtrToStringAnsi has been working consistently for me. I assumed it was because GError was simply encoded in ANSI, but in hindsight this doesn't make any sense, since Unicode should work too. Any ideas why this could be?

Reply:
Interesting, I just did a test with fedora and for me both methods work. So I think we should use the "Ansi" method.

On another sidenote: I saw your work for heyena and I think it is great :+1: . There is a lot to do for gir.core and with the samples running we can check how to improve the generation process. But with hyena we have some real world project which we can bring to live. For now I will focus on gir.core as there is more or less nothing working here :wink:. But I think we can profit from your early work and tests with hyena as you will discover real world problems using gir.core. If you have some difficulties or need support just ask I'm willing to help :rocket: